### PR TITLE
Reduce motion of Terra banners [BW-755][hackathon]

### PR DESCRIPTION
**Pull request by aednichols**
Replace slide-in animation with a subtle fade. This is an [accessibility measure](https://a11y-101.com/development/reduced-motion) that helps users who have trouble concentrating in the face of distraction (like me!).

*Before:*

Jul-14-2021 15-37-05

*After:*

Jul-14-2021 15-37-12

You may have come across this similar setting in iOS:

IMG_5103

In this case I did not elect to make it a setting because I don't think the absence of sliding will impair anyone from noticing the banners, and it would be hard to make the setting discoverable.

### CODE REVIEW

**Review by s-rubenstein — APPROVED** [review]
I really like this change from a visual clutter perspective. 

Just to check, have you run this by UX?

File: src/libs/notifications.js
Comment (s-rubenstein) [inline src/libs/notifications.js:151]: I might leave the animationOut as a slide? Just since that one (I think) needs to be initiated by the user, a harder out animation might be nice for stronger immediate feedback, but it would disrupt the symmetry there. I would defer to UX.

**Review by salonishah11 — APPROVED** [review]
As long as UX is happy, it LGTM 👍

**Comment by aednichols** [comment]
[Joy says 💯](https://broadinstitute.slack.com/archives/C01EHNUM73R/p1626795251152900?thread_ts=1626291845.151600&cid=C01EHNUM73R)

**Comment by aednichols** [comment]
Adding @breilly2 as the most proximate Terra code owner

**Comment by aednichols** [comment]
Apologies for putting @breilly2 on the spot, have learned that he is no longer a code owner & removed from review list (but if you have feedback anyway I always welcome it!)

**Review by dmohs — APPROVED** [review]
Thank you for caring about stuff like this, Adam.

**Pull request merged**
